scripts/scripts.js

```diff
@@ -43,6 +69,12 @@ function buildAutoBlocks(main) {
     // eslint-disable-next-line no-console
     console.error('Auto Blocking failed', error);
```

Comment:
You are missing some important things here.
Line 70 should look like this:

function buildAutoBlocks(main) {
  try {
    buildAutoblogBlock(main);
  } catch (error) {
    // eslint-disable-next-line no-console
    console.error('Auto Blocking failed', error);
  }

Reply:
@helms-charity  - not sure what important thing u r referring here. Is there anything apart from Camel Casing that you are referring here? The Code is already within the function which is within the function buildAutoBlocks(main) function. 
 
function buildAutoBlocks(main) {
  try {
    buildHeroBlock(main);
  } catch (error) {
    // eslint-disable-next-line no-console
    console.error('Auto Blocking failed', error);
  }
  try {
    buildAutoblogBlock(main);
  } catch (error) {
    // eslint-disable-next-line no-console
    console.error('Auto Blocking failed', error);
  }
}

Reply:
You are absolutely right to be confused -- I neglected to notice that Github had collapsed part of the code, O thought it was missing! My bad!!